Assets/Tests/Demo/scripts/verify-replay-via-cli.sh

```
@@ -0,0 +1,205 @@
+#
+# Usage: sh verify-replay-via-cli.sh [--project-path <path>]
+#
+# Prerequisites:
+#   - Unity Editor running with InputReplayVerificationScene loaded
+#   - PlayMode is NOT running (script starts it)
+
+set -e
+
+PROJECT_OPTS=""
+if [ "$1" = "--project-path" ] && [ -n "$2" ]; then
+    PROJECT_OPTS="--project-path \"$2\""
```

Shellcheck (0.11.0)

[warning] Quotes/backslashes will be treated literally. Rewrite using set/"$@" or functions.

(SC2089)